aten/src/ATen/native/native_functions.yaml

```diff
@@ -10897,6 +10897,21 @@
     CUDA: foreach_tensor_zero_cuda_
   autogen: _foreach_zero, _foreach_zero.out
 
+- func: _foreach_sign(Tensor[] self) -> Tensor[]
```

Comment:
This is a nit but I had organized the _foreach_* ops here alphabetically beyond the main add/sub/mul/divs and it would be nice to have this be between sigmoid and sin 😁